core/src/martian/schema.cljc

```diff
@@ -85,21 +85,26 @@
               {(cond-> (keyword name)
                  (not required)
                  s/optional-key)
-               (make-schema definitions param)}))
+               (make-schema definitions {} param)}))
        (into {})))
 
-(defn- resolve-ref [definitions ref]
-  (some->> ref
-           (re-find #"#/definitions/(.*)")
-           second
-           keyword
-           definitions))
+(defn- resolve-ref [definitions parameters ref]
+  (if-let [d (re-find #"#/definitions/(.*)" ref)]
```

Comment:
Then on this line you could change the regex to be
`(let [[_ category k] (re-find #"#/(definitions|parameters)/(.*)" ref)]
  (get-in ref-lookup [(keyword category) (keyword k)]))`

I'm not sure if 'category' is the best word to describe it.

Reply:
Please review the most recent comment.  Thank you.